src/fonts/Roboto/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Can we have a `/ACKNOWLEDGMENTS.md` file that lists all products and the paths of their respective licenses (e.g., this one)

Reply:
Good thinking. Another think I didn't think about was that this file isn't even included in the final build (and the license states that it must be included with redistributions and the docker image _is_ a redist).

Reply:
> this file isn't even included in the final build

Has this changed?